Carthage.xcodeproj/xcshareddata/xcschemes/xcproj_iOS.xcscheme

```diff
@@ -1,28 +1,93 @@
-<?xml version="1.0" encoding="utf-8" standalone="no"?>
-<Scheme LastUpgradeVersion="0900" version="1.3">
-	<AnalyzeAction buildConfiguration="Debug" />
-	<ArchiveAction buildConfiguration="Release" revealArchiveInOrganizer="YES" />
-	<TestAction selectedDebuggerIdentifier="Xcode.DebuggerFoundation.Debugger.LLDB" selectedLauncherIdentifier="Xcode.DebuggerFoundation.Launcher.LLDB" shouldUseLaunchSchemeArgsEnv="YES" buildConfiguration="Debug">
-		<Testables />
-		<MacroExpansion>
-			<BuildableReference BlueprintIdentifier="NT4163490801" ReferencedContainer="container:Carthage.xcodeproj" BuildableName="xcproj_iOS.framework" BlueprintName="xcproj_iOS" BuildableIdentifier="primary" />
-		</MacroExpansion>
-	</TestAction>
-	<ProfileAction savedToolIdentifier="" useCustomWorkingDirectory="NO" shouldUseLaunchSchemeArgsEnv="YES" buildConfiguration="Release" debugDocumentVersioning="YES">
-		<BuildableProductRunnable runnableDebuggingMode="0">
-			<BuildableReference BlueprintIdentifier="NT4163490801" ReferencedContainer="container:Carthage.xcodeproj" BuildableName="xcproj_iOS.framework" BlueprintName="xcproj_iOS" BuildableIdentifier="primary" />
-		</BuildableProductRunnable>
-	</ProfileAction>
-	<BuildAction parallelizeBuildables="YES" buildImplicitDependencies="YES">
-		<BuildActionEntries>
-			<BuildActionEntry buildForArchiving="YES" buildForTesting="YES" buildForRunning="YES" buildForProfiling="YES" buildForAnalyzing="YES">
-				<BuildableReference BlueprintIdentifier="NT4163490801" ReferencedContainer="container:Carthage.xcodeproj" BuildableName="xcproj_iOS.framework" BlueprintName="xcproj_iOS" BuildableIdentifier="primary" />
-			</BuildActionEntry>
-		</BuildActionEntries>
-	</BuildAction>
-	<LaunchAction selectedDebuggerIdentifier="Xcode.DebuggerFoundation.Debugger.LLDB" selectedLauncherIdentifier="Xcode.DebuggerFoundation.Launcher.LLDB" launchStyle="0" buildConfiguration="Debug" debugServiceExtension="internal" ignoresPersistentStateOnLaunch="NO" useCustomWorkingDirectory="NO" allowLocationSimulation="YES" debugDocumentVersioning="YES">
-		<BuildableProductRunnable runnableDebuggingMode="0">
-			<BuildableReference BlueprintIdentifier="NT4163490801" ReferencedContainer="container:Carthage.xcodeproj" BuildableName="xcproj_iOS.framework" BlueprintName="xcproj_iOS" BuildableIdentifier="primary" />
-		</BuildableProductRunnable>
-	</LaunchAction>
-</Scheme>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Were these schemes committed after opening the carthage project? As the formatting has changed

Reply:
Undone, thanks!